scripts/zones/Waughroon_Shrine/IDs.lua

```diff
@@ -84,6 +84,13 @@ zones[xi.zone.WAUGHROON_SHRINE] =
             17367285,
         },
     },
+
+    royalJellyQueens =
```

Comment:
These can be looked up at runtime with zone:queryEntitiesByName or GetFirstID taking advantage of the fact that they're all offset from eachother by 10. Or drop in GetTableOfIDs for a one-line replacement

Reply:
done